src/vmm/src/vmm_config/mod.rs

```diff
@@ -194,14 +175,14 @@ mod tests {
     }
 
     #[test]
-    fn test_log_writer() {
+    fn test_fifo_line_writer() {
         let log_file_temp =
             TempFile::new().expect("Failed to create temporary output logging file.");
         let good_file = log_file_temp.as_path().to_path_buf();
-        let res = Writer::new(good_file);
-        assert!(res.is_ok());
+        let maybe_fifo = open_fifo(&good_file);
```

Comment:
nit: `maybe_*` is usually used in the async context. `fifo` should suffice here.

Reply:
I used `maybe_` because this returns an option. It can contain something or it can be `None`. I think I saw this in the rust examples somewhere, but I don't remember where exactly.